src/ui/ExpandableSection/ExpandableSection.tsx

```diff
@@ -0,0 +1,67 @@
+import * as Collapsible from '@radix-ui/react-collapsible'
+import cs from 'classnames'
```

Comment:
Should use new `cn` wrapper instead of `cs`

Reply:
It's a drop in replacement

Reply:
Updated. Why did we replace cs though?

Reply:
`cn` is a mix of classnames, and another utility called `tailwind-merge`, `tailwind-merge` will remove duplicate and potentially conflicting classnames for us, which has caused some issues beforehand.